firmware/application/apps/ui_recon.cpp

```diff
@@ -1102,6 +1110,10 @@ void ReconView::on_statistics_update(const ChannelStatistics& statistics) {
                     // contents of a possible recon_start_recording(), but not yet since it's only called once
                     if (auto_record_locked && !is_recording) {
                         button_audio_app.set_style(&Styles::red);
+                        if (field_mode.selected_index_value() == SPEC_MODULATION)
```

Comment:
Would it be interesting to set the bandwidth to match (like we did for audio)?

Reply:
We can, but I don't know how yet ^^

Reply:
Could you do it in another PR ?

Reply:
Does Recon see bandwidth in Spec mode?

Reply:
I think yes